pkg/appgw/frontend_listeners.go

```diff
@@ -42,6 +42,7 @@ func (c *appGwConfigBuilder) getListeners(ingressList []*v1beta1.Ingress) (*[]n.
 func (c *appGwConfigBuilder) getListenerConfigs(ingressList []*v1beta1.Ingress) map[listenerIdentifier]listenerAzConfig {
 	allListeners := make(map[listenerIdentifier]listenerAzConfig)
 	for _, ingress := range ingressList {
+		glog.V(5).Infof("Processing Rules for Ingress: %s/%s", ingress.Namespace, ingress.Name)
```

Comment:
We should raise an event if two namespaces are using the same hostname port and have different certificates.
We should proceed by using the secret but a warning to resolve this conflict will help.

Reply:
I left a `TODO` and will work on this in a fast-follow PR - would love to unblock this PR